http/java/okhttp/lib/src/main/java/com/microsoft/kiota/http/middleware/options/TelemetryUtilities.java

```diff
@@ -0,0 +1,100 @@
+package com.microsoft.kiota.http.middleware.options;
+
+import java.util.Objects;
+import java.util.UUID;
+import java.lang.reflect.Field;
+
+import javax.annotation.Nonnull;
+
+public class TelemetryUtilities {
+    
+    /**
+     * The Client Request ID for use
+     */
+    private String clientRequestId;
+    /**
+     * Current SDK version
+     */
+    public static final String VERSION = ""; //Set This Upon Release
```

Comment:
move this back to graph core

Reply:
I'll move this back to core, TelemetryHandlerOption.java and TelemetryHandler.java will stay.